src/Interfaces/Authenticatable.php

```diff
@@ -12,10 +12,10 @@ public function getAuthIdColumn(): string;
 
     /**
      * Returns the unique identifier of
-     * the object for authentication purposes.
+     * the User object for authentication purposes.
      * Typically the user's id.
      *
-     * @return mixed
+     * @return int|string|null
```

Comment:
In what scenario would this be `null`?

Reply:
Agreed. I like valid state objects.

In this case, the User object should have vaild ID, and if not it is a bug.

Done.

Reply:
Done!